ext/drreg/drreg.h

```diff
@@ -274,6 +277,9 @@ DR_EXPORT
  * If called during drmgr's insertion phase, \p where must be the
  * current application instruction.
  *
+ * TODO i#3823: Support multi-phase use. This will require adding support
```

Comment:
Any thoughts on priority for this? I'm leaning towards adding this support after this PR and the scatter/gather one are done.

Reply:
Since I see drx_expand_scatter_gather calling drreg_reserve_aflags, it does seem like this is high-ish priority, right?

Reply:
I think supporting multi-phase spill for aflags requires some more work. I have just started looking into it, but unlike other regs, aflags seem to be tied to `AFLAGS_SLOT`.

If you agree that this PR is not in an inconsistent state without the aflag support, I'd prefer to submit this first and work on the aflag support in a new PR. I'm feeling there's too many things in-flight with this PR and PR#4863 still pending.

Reply:
Agreed on splitting: I meant it's high-ish priority in general, not that it needs to be tied to this PR.

Reply:
Thanks. Changed this PR's title to mention GPRs specifically and added a new TODO on i#2985 for aflags.